src/components/svgicon/SVGIcon.tsx

```diff
@@ -28,7 +28,7 @@ const VIEW_BOX = "0 0 24 24";
 
 export type SVGIconProps = React.HTMLAttributes<HTMLOrSVGElement> & {
   name: PathName;
-  color?: SingleColors;
+  color?: string;
```

Comment:
This should add autocomplete option to `color` prop, while still allowing to pass any string.

type AnyString = Record<never, never> & string;
type SingleColors = "#454545" | "black" | "white";
type SvgColor = AnyString | SingleColors;

```suggestion
  color?: SvgColor;
```